perf_benchmark/benchmark_service.textproto

```diff
@@ -161,6 +161,13 @@ types {
     type_url: "type.googleapis.com/google.protobuf.Value"
     json_name: "value"
   }
+  options {
```

Comment:
This change should be submitted by your other pr.  Why is it still here?

Reply:
Ohh sorry for the confusion. We have two textprotos - one for the original transcoder unit test, and the other just for the benchmark. This is because We have different proto definitions for the unit tests and benchmarks.

PR #78 changed the `test/testdata/bookstore_service.ob.txt` for the unit test, and this change is for the benchmark textproto.